pom.xml

```diff
@@ -38,8 +38,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>21</source>
-                    <target>21</target>
+                    <source>19</source>
```

Comment:
Why downgrade?

Reply:
Sorry, my mistake. The case of error is my inattention.
Fixed.